appveyor.yml

```
@@ -1,6 +1,6 @@
 environment:
   matrix:
-  - julia_version: 0.7
+  - julia_version: 1.3
```

```suggestion
  - julia_version: 1.0
  - julia_version: 1.3
```

need to test the LTS